crates/shared/src/price_estimation.rs

```diff
@@ -486,6 +487,8 @@ pub struct Estimate {
     pub gas: u64,
     /// Address of the solver that provided the quote.
     pub solver: H160,
+    /// Did we verify the correctnes of this estimates properties?
```

Comment:
`correctness`

Reply:
Oh, no. Not at the base of my PR stack! :sweat_smile: